apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.html.ts

```diff
@@ -18,53 +24,57 @@ export function InstructorAiGenerateDraftEditor({
   question: Question;
   variantId?: string | undefined;
 }) {
-  return PageLayout({
-    resLocals,
-    pageTitle: 'Generate question with AI',
-    navContext: {
-      type: 'instructor',
-      page: 'course_admin',
-      subPage: 'questions',
-    },
-    options: {
-      hxExt: 'loading-states',
-    },
-    headContent: html`
-      ${compiledScriptTag('question.ts')}
-      <script defer src="${nodeModulesAssetPath('mathjax/es5/startup.js')}"></script>
-    `,
-    content: html`
-      <div class="mb-3">
-        <a href="${resLocals.urlPrefix}/ai_generate_question_drafts" class="btn btn-sm btn-primary">
-          <i class="fa fa-arrow-left" aria-hidden="true"></i>
-          Back to draft questions
-        </a>
-      </div>
-      <div class="card">
-        <div
-          class="card-header bg-primary text-white d-flex align-items-center justify-content-between"
-        >
-          Generate question with AI
-
-          <div>
-            <button
-              type="button"
-              class="btn btn-sm btn-light"
-              data-toggle="modal"
-              data-target="#finalizeModal"
-            >
-              <i class="fa fa-check" aria-hidden="true"></i>
-              Finalize question
-            </button>
-          </div>
-        </div>
-        <div class="card-body">
-          <div class="container">
-            <div class="row">
-              <div class="col-12 col-md-6">
-                <div>
-                  ${PromptHistory({ prompts, urlPrefix: resLocals.urlPrefix })}
+  // This page has a very custom layout, so we don't use the usual `PageLayout`
+  // component here. If we start building other similar pages, we might want to
+  // teach `PageLayout` how to render this kind of layout.
```

Comment:
We'll probably want to do this regardless once we start working on new navigation behind a feature flag. I'm kicking that can down the road for now.